molgenis-platform-integration-tests/pom.xml

```diff
@@ -9,12 +9,162 @@
+                            </dependency>
+                        </dependencies>
+                        <configuration>
+                            <driver>org.postgresql.Driver</driver>
+                            <url>jdbc:postgresql://localhost/</url>
+                            <username>${it_db_user}</username>
+                            <password>${it_db_password}</password>
+                            <forceMojoExecution>true</forceMojoExecution>
+                            <onError>continue</onError>
+                            <autocommit>true</autocommit>
+                            <skip>${skipTests}</skip>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>create postgres integration database</id>
+                                <goals>
+                                    <goal>execute</goal>
+                                </goals>
+                                <phase>pre-integration-test</phase>
+                                <configuration>
+                                    <sqlCommand>create database ${it_db_name};</sqlCommand>
+                                </configuration>
+                            </execution>
+                            <execution>
+                                <id>drop postgres integration database</id>
+                                <goals>
+                                    <goal>execute</goal>
+                                </goals>
+                                <phase>post-integration-test</phase>
+                                <configuration>
+                                    <sqlCommand>drop database ${it_db_name};</sqlCommand>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <!-- Profile create-it-es creates new elasticsearch instance and drops it after the test -->
+        <profile>
+            <id>create-it-es</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <maven.build.timestamp.format>yyMMddHHmmssSSS</maven.build.timestamp.format>
+                <elasticsearch.cluster.name>molgenis${maven.build.timestamp}</elasticsearch.cluster.name>
+                <elasticsearch.transport.addresses>localhost:${elasticsearch.tcp.port}
+                </elasticsearch.transport.addresses>
+            </properties>
+            <build>
+                <plugins>
+                    <!-- Reserve tcp and http ports for elasticsearch -->
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>build-helper-maven-plugin</artifactId>
+                        <version>3.0.0</version>
```

Comment:
Master pom is using different build-helper-maven-plugin version. Can we use the same version?